config/locales/user_mailer/en.yml

```diff
@@ -44,6 +44,18 @@ en:
         %{help_link} or %{contact_link}.
       intro: The email address for your %{app} account has been changed.
     help_link_text: Help Center
+    letter_expired:
+      help: ''
+      info: The letter we sent to your address has a code that expired and can no
+        longer be used to complete the verification process.  Please sign into %{link}
+        and follow instructions.
```

Comment:
I am no wordsmith, but "please follow instructions (this time)" sounds condescending and alarmist. Could we just tell the user what the next step is here. "To verify your identity you will need to sign into Login.gov and choose to send another letter".